chaoslib/litmus/spring-boot-chaos/lib/spring-boot-chaos.go

```diff
@@ -0,0 +1,329 @@
+	if err != nil {
+		return err
+	}
+
+	if resp.StatusCode != 200 {
+		return errors.Errorf("failed to disable chaos monkey endpoint on pod %v (status: %v)", pod.Name, resp.StatusCode)
+	}
+
+	return nil
+}
+
+// injectChaosInSerialMode delete the target application pods serial mode(one by one)
+func injectChaosInSerialMode(experimentsDetails *experimentTypes.ExperimentDetails, clients clients.ClientSets, chaosDetails *types.ChaosDetails, eventsDetails *types.EventDetails, resultDetails *types.ResultDetails) error {
+
+	// run the probes during chaos
+	if len(resultDetails.ProbeDetails) != 0 {
+		if err := probe.RunProbes(chaosDetails, clients, resultDetails, "DuringChaos", eventsDetails); err != nil {
+			return err
+		}
+	}
+
+	// signChan channel is used to transmit signal notifications.
+	signChan := make(chan os.Signal, 1)
+	// Catch and relay certain signal(s) to signChan channel.
+	signal.Notify(signChan, os.Interrupt, syscall.SIGTERM)
+
+	var endTime <-chan time.Time
+	timeDelay := time.Duration(experimentsDetails.ChaosDuration) * time.Second
+
+	select {
+	case <-signChan:
+		// stopping the chaos execution, if abort signal received
+		time.Sleep(10 * time.Second)
+		os.Exit(0)
+	default:
+		for _, pod := range experimentsDetails.TargetPodList.Items {
+			if experimentsDetails.EngineName != "" {
+				msg := "Injecting " + experimentsDetails.ExperimentName + " chaos on " + pod.Name + " pod"
+				types.SetEngineEventAttributes(eventsDetails, types.ChaosInject, msg, "Normal", chaosDetails)
+				_ = events.GenerateEvents(eventsDetails, clients, chaosDetails, "ChaosEngine")
+			}
+
+			log.InfoWithValues("[Chaos]: The Target application details", logrus.Fields{
+				"Target Pod": pod.Name,
+			})
+
+			if err := setChaosMonkeyWatchers(experimentsDetails.ChaosMonkeyPort, experimentsDetails.ChaosMonkeyPath, experimentsDetails.ChaosMonkeyWatchers, pod); err != nil {
+				log.Errorf("[Chaos]: Failed to set watchers, err: %v ", err)
+				return err
+			}
+
+			if err := setChaosMonkeyAssault(experimentsDetails.ChaosMonkeyPort, experimentsDetails.ChaosMonkeyPath, experimentsDetails.ChaosMonkeyAssault, pod); err != nil {
+				log.Errorf("[Chaos]: Failed to set assault, err: %v ", err)
+				return err
+			}
+
+			if err := enableChaosMonkey(experimentsDetails.ChaosMonkeyPort, experimentsDetails.ChaosMonkeyPath, pod); err != nil {
+				log.Errorf("[Chaos]: Failed to enable chaos, err: %v ", err)
+				return err
+			}
+			common.SetTargets(pod.Name, "injected", "pod", chaosDetails)
+
+			log.Infof("[Chaos]:Waiting for: %vs", experimentsDetails.ChaosDuration)
+
+			endTime = time.After(timeDelay)
+		loop:
+			for {
+				select {
+				case <-signChan:
+					log.Info("[Chaos]: Revert Started")
+					if err := disableChaosMonkey(experimentsDetails.ChaosMonkeyPort, experimentsDetails.ChaosMonkeyPath, pod); err != nil {
+						log.Errorf("Error in disable chaos monkey, err: %v", err)
```

Comment:
```suggestion
						log.Errorf("Error in disabling chaos monkey, err: %v", err)
```
Can you edit this at other places as well?